connectors/aws-dynamo-db/pom.xml

```
@@ -0,0 +1,80 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.camunda.connector</groupId>
+        <artifactId>connector-function-parent</artifactId>
+        <version>0.18.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <name>connector-aws-dynamo-db</name>
+    <description>Camunda Connector AWS DynamoDB function</description>
+    <artifactId>connector-aws-dynamo-db</artifactId>
+    <packaging>jar</packaging>
+    <properties>
+        <license.inlineheader>Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+            under one or more contributor license agreements. See the NOTICE file
+            distributed with this work for additional information regarding copyright
+            ownership. Camunda licenses this file to you under the Apache License,
+            Version 2.0; you may not use this file except in compliance with the License.
+            You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+            Unless required by applicable law or agreed to in writing, software
+            distributed under the License is distributed on an "AS IS" BASIS,
+            WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+            See the License for the specific language governing permissions and
+            limitations under the License.</license.inlineheader>
+    </properties>
+
+
+    <dependencies>
+
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.danilopianini</groupId>
+            <artifactId>gson-extras</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-dynamodb</artifactId>
+            <version>1.12.440</version>
```

You need to reuse supplied AWS lib, ref: https://github.com/camunda/connectors-bundle/commit/764fedc6fae9693d2279a47ff71c49f4fb63366d

```suggestion
            <version>${aws-java-sdk}</version>
```